htdocs/mrp/class/mo.class.php

```diff
@@ -655,6 +655,8 @@ public function updateProduction(User $user, $notrigger = true)
 							$moline = new MoLine($this->db);
 
 							$moline->fk_mo = $this->id;
+							$moline->origin_id = $line->id;
+							$moline->origin_type = 'bom';
```

Comment:
Is it id of BOM or id of line in a BOM ?
If id of line in a BOM, the origin_type = 'bom_line' is better.

Reply:
Should I do $moline->origin_type = $line->element; ?